integration_tests/mockApis/auth.ts

```diff
@@ -122,6 +131,16 @@ export default {
   getSignInUrl,
   stubAuthPing: ping,
   stubAuthToken: token,
-  stubSignIn: (role: string): Promise<[Response, Response, Response, Response, Response, Response]> =>
-    Promise.all([favicon(), redirect(), signOut(), manageDetails(), token(role), tokenVerification.stubVerifyToken()]),
+  stubAuthManageDetails: manageDetails,
+  stubSignIn: (
+    userToken: UserToken = { roles: ['ROLE_MANAGE_PRISON_VISITS'] },
+  ): Promise<[Response, Response, Response, Response, Response, Response]> =>
+    Promise.all([
+      favicon(),
+      redirect(),
+      signOut(),
+      token(userToken),
+      tokenVerification.stubVerifyToken(),
+      stubComponents(),
```

Comment:
Adding stub for DPS components to `stubSignIn` to avoid having to add it to every test.